uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -247,55 +247,50 @@ sub IsCandidateInfoValid {
         ########################################################
         #1) Exlcude files starting with . (and ._ as a result)##
         #including the .DS_Store file###########################
-        #2) Check to see if the file is of type DICOM###########
+        #2) Exclude files that are not of type DICOM ###########
         #3) Check to see if the header matches the patient-name#
         ########################################################
-        if ( (basename($_) =~ /^\./)) {
+        if ( (basename($_) =~ /^\./) || !$this->isDicom($_)) {
             $cmd = "rm " . ($_);
             print "\n $cmd \n";
             system($cmd);
```

Comment:
I am worried that you mind end up erasing DICOM files that are not of type DICOM or files that people might still want to keep in the archive for some reason.

I would have a tendency to only execute the command for files that starts with .. What do you think?

I don't have a strong opinion about this though. Just thought I would raise this since it crossed my mind.

Reply:
(this comment was supposed to be part of my previous review...)

Reply:
This is a very good point. It happened in the Quad project with Rick. There were some non-dicom files that contained some metadata or something. I don't think that these non-dicom files should be removed.

Reply:
If you have come across projects that needed to keep files in the archive even though they were not DICOM files, then yeah we should leave the archive as is and not delete anything (maybe only the files that start with ".") 

@cmadjar What do you mean when you say "...end up erasing DICOM files that are not of type DICOM"?

Reply:
@nicolasbrossard I think someone (can't remember which project) at some point that came up with a DICOM dataset but the files were not of type DICOM for some reason when typing in the terminal `file dicom.dcm`.